src/main/resources/templates/search-result.html

```diff
@@ -10,9 +10,9 @@
 <title>Gas Mileage Calculator</title>
 </head>
 <body>
+<div th:replace="fragments/header :: header"></div>
 <div class="container">
-<h2>Gas Mileage Calculator</h2>
-<h4>Search Result</h1>
+<h4>Search Result</h4>
```

Comment:
Replace the header component in each page with a reference to the header fragment

Reply:
Much easier with head fragments so you don't have to type the header in every time.

Reply:
I agree, this way it makes life easier.